rct229/rules/section21/section21rule10.py

```diff
@@ -0,0 +1,116 @@
+from rct229.data.schema_enums import schema_enums
+from rct229.rule_engine.rule_base import RuleDefinitionBase
+from rct229.rule_engine.rule_list_indexed_base import RuleDefinitionListIndexedBase
+from rct229.rule_engine.user_baseline_proposed_vals import UserBaselineProposedVals
+from rct229.ruleset_functions.baseline_systems.baseline_system_util import HVAC_SYS
+from rct229.ruleset_functions.get_baseline_system_types import get_baseline_system_types
+from rct229.ruleset_functions.get_hw_loop_zone_list_w_area_dict import (
+    get_hw_loop_zone_list_w_area,
+)
+from rct229.schema.config import ureg
+from rct229.utils.assertions import getattr_
+
+APPLICABLE_SYS_TYPES = [
+    HVAC_SYS.SYS_1,
+    HVAC_SYS.SYS_5,
+    HVAC_SYS.SYS_7,
+    HVAC_SYS.SYS_11_2,
+    HVAC_SYS.SYS_12,
+    HVAC_SYS.SYS_1A,
+    HVAC_SYS.SYS_7A,
+    HVAC_SYS.SYS_11_2A,
+    HVAC_SYS.SYS_12A,
+    HVAC_SYS.SYS_1B,
+    HVAC_SYS.SYS_3B,
+    HVAC_SYS.SYS_5B,
+    HVAC_SYS.SYS_6B,
+    HVAC_SYS.SYS_7B,
+    HVAC_SYS.SYS_8B,
+    HVAC_SYS.SYS_9B,
+    HVAC_SYS.SYS_11_1B,
+    HVAC_SYS.SYS_12B,
+    HVAC_SYS.SYS_1C,
+    HVAC_SYS.SYS_3C,
+    HVAC_SYS.SYS_7C,
+    HVAC_SYS.SYS_11_1C,
+    HVAC_SYS.SYS_12C,
+]
+
+PUMP_SPEED_CONTROL = schema_enums["PumpSpeedControlOptions"]
+PUMP_CONFIGURATION_THRESHOLD = 120000 * ureg("ft2")
```

Comment:
As of python 3.6, we can write a number literal using an underscore as the separator.  So you could write:
```
PUMP_CONFIGURATION_THRESHOLD = 120_000 * ureg("ft2")
```
This has the advantage of being more human readable.